front/app/components/IdeaCards/IdeasWithFiltersSidebar/index.tsx

```diff
@@ -122,9 +122,12 @@ const ContentRight = styled.div<{ filterColumnWidth: number }>`
   justify-content: flex-start;
   align-self: flex-start;
   margin-left: ${gapWidth}px;
-  position: relative;
+  max-height: calc(100vh - 120px);
   position: sticky;
```

Comment:
Once we use this component within the Project page we'll have to calculate this differently based on whether there is also a sticky CTA bar being shown, but I thought we could tackle that in a separate PR when we get there?